spec/views/layouts/application.html.haml_spec.rb

```diff
@@ -1,85 +1,101 @@
-          active: false,
-        },
-        {
-          name: "First Level Item 2",
-          url: "ressource/2",
-          active: true,
-        },
-        {
-          name: "First Level Item 3",
-          url: "ressource/3",
-          active: true,
-        }
-      ], [
-        {
-          name: "Second Level Item 1",
-          url: "/ressource/1/one",
-          active: true
-        },
+      def current_user
+        FactoryGirl.build(:user, name: 'Hans Schwanitz')
+      end
+    end
+
+    before do
+      assign(:report , FactoryGirl.create(:report))
+      assign(:subnav_items, [
+        [
+          {
+            name: "First Level Item 1",
+            url: "/ressource/1",
+            active: false,
+          },
+          {
+            name: "First Level Item 2",
+            url: "ressource/2",
+            active: true,
+          },
+          {
+            name: "First Level Item 3",
+            url: "ressource/3",
+            active: true,
+          }
+        ], [
+          {
+            name: "Second Level Item 1",
+            url: "/ressource/1/one",
+            active: true
+          },
+          {
+            name: "Second Level Item 2",
+            url: "/ressource/2/two",
+            active: false
+          }
+        ]
+      ])
+
+      assign(:primary_action, {
+        name: 'Primary Call To Action',
+        url: '/ressource/new'
+      })
+
+      assign(:secondary_actions, [
         {
-          name: "Second Level Item 2",
-          url: "/ressource/2/two",
-          active: false
+          name: 'Secondary Action 1',
+          url: '/do/this'
+        }, {
+          name: 'Secondary Action 2',
+          url: '/and/that'
         }
-      ]
-    ])
-
-    assign(:primary_action, {
-      name: 'Primary Call To Action',
-      url: '/ressource/new'
-    })
+      ])
 
-    assign(:secondary_actions, [
-      {
-        name: 'Secondary Action 1',
-        url: '/do/this'
-      }, {
-        name: 'Secondary Action 2',
-        url: '/and/that'
-      }
-    ])
+    end
+    
+    let(:parsed) do
+      render
+      Capybara.string(rendered)
+    end
 
-    render
-    @parsed = Capybara.string(rendered)
-  end
 
-  context 'renders a breadcrumb menu' do
+    context 'renders a breadcrumb menu' do
 
-    it 'has multiple levels' do
-      expect(@parsed).to have_css('.subnav__select', count: 2)
-    end
+      it 'has multiple levels' do
+        expect(parsed).to have_css('.subnav__select', count: 2)
+      end
 
-    it 'has multiple select options with respective labels and urls' do
-      expect(@parsed).to have_css('.subnav__breadcrumb__item:nth-child(1) select option', count: 3)
-      expect(@parsed).to have_css('.subnav__breadcrumb__item:nth-child(2) select option', count: 2)
+      it 'has multiple select options with respective labels and urls' do
+        expect(parsed).to have_css('.subnav__breadcrumb__item:nth-child(2) select option', count: 3)
```

Comment:
hey @tillprochaska can you have a look into this before we merge it? Why do I have to increment the `nth-child` here?

Reply:
@roschaefer: I did not take a closer look at this. but in #623 I had to decrement the index again to make tests work. Either I completely overlooked something or this is an unintended behavior of PhantomJS…